src/components/Menu/Menu.tsx

```diff
@@ -18,7 +18,7 @@ interface Props {
 
 export type MenuProps = PropsWithChildren<Props>;
 
-const CONTAINER_BASE_CLASSES = 'tw-relative tw-bg-base tw-rounded tw-py-2 tw-shadow-mid tw-w-max';
+const CONTAINER_BASE_CLASSES = 'tw-relative tw-bg-base tw-rounded tw-py-2 tw-shadow-mid tw-min-w-max';
```

Comment:
Just saw a problem with the `Menu` z-index I let pass during the development. You can see on the component Docs page on Storybook by opening the Basic Menu With Links. It will be behind the Menu Without Trigger section.
Adding the class `tw-z-[120000]` to the base classes will solve the issue (this is the same class applied on the `Tooltip` component)